pages/releases/4.3.html

```diff
@@ -0,0 +1,2360 @@
+						</div>
+						<div class="release-card-media">
+							<video class="release-card-video lazy" playsinline disablepictureinpicture autoplay loop muted
+								poster="/storage/releases/4.3/images/body-face-tracking.webp">
+								<source data-src="/storage/releases/4.3/video/body-face-tracking.webm"
+									type="video/webm" />
+							</video>
+						</div>
+					</div>
+				</div> <!-- #xr-standardized-hand-body-face-tracking -->
+
+				<div id="xr-performance-stability-improvements" class="release-card span-3">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#xr-performance-stability-improvements">
+										Performance and stability improvements
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										Due to countless improvements made to this part of the engine, XR applications are now more stable and performant.
+									</p>
+									<p>
+										Highlights include:
+									</p>
+									<ul>
+										<li>
+											improved timing of tracking data
+										</li>
+										<li>
+											XR is now thread safe, allowing you to move rendering onto a separate thread
+										</li>
+										<li>
+											foveated rendering has been tweaked on the <em>Mobile</em> and <em>Forward+</em>
+											renderers
+										</li>
+										<li>
+											glasses-based AR & passthrough-based AR have been unified
+										</li>
+									</ul>
+								</div>
+								<div class="c-link"
+									data-contributors="BastiaanOlij,dsnopek"
+									data-read-more="https://github.com/godotengine/godot/pulls?q=87630+89734+89894">
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #xr-performance-stability-improvements -->
+
+				<div id="xr-meta-xr-simulator-support" class="release-card span media-bottom">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#xr-meta-xr-simulator-support">
+										Meta XR Simulator support
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										One complication of developing XR games is having to repeatedly test your
+										application on a headset, especially if your headset is a standalone VR
+										unit (read: not connected to a computer).
+									</p>
+									<p>
+										The Meta XR Simulator is a free tool to test your XR application outside
+										of the headset, but only works if you are targeting the Quest series models.
+										After downloading and installing it on your machine, you can now configure
+										it to work directly within your Godot editor. This feature is limited to
+										Windows and macOS because the simulator itself currently does not support
+										Linux.
```

Comment:
```suggestion
									<p>
										The Meta XR Simulator is a free tool to test your XR application in a simulated Quest headset instead.
										After downloading and installing it on your machine, you can now configure
										it to work directly within your Godot editor. This feature is limited to
										Windows and macOS because the simulator itself currently does not support
										Linux.
```